lib/scss/components/news-vertical-teaser/_news-vertical-teaser.scss

```diff
@@ -0,0 +1,39 @@
+@charset "UTF-8";
+
+.su-card {
```

Comment:
Components should have a wrapper class that is named the same as the component. eg: `.su-news-vertical-teaser`

Reply:
Fixed this in the new commit.